### ISSUE TITLE
Trix editor widget

### ISSUE THREAD

**Issue by joff13** [viewflow/viewflow#454]
Hi 

To make Trix widget work, we had to copy the task.html and add  at the begging as follow

after the first 2 lines
```
{% extends request.resolver_match.app.base_template_name|default:'viewflow/workflow/base_page.html' %}
{% load i18n viewflow workflow %}
```
new insert
```
{# Copied from vievflow/form.html to include css/js defined in fields Media class #}
{% block extrahead %}
  {{ block.super }}
  {{ form.media }}
{% endblock %}
```
existing line
```
{% block content %}{% with process=request.activation.process task=request.activation.task %}
```

Am I missing something or it's a bug ?
Regards
C

**Comment by kmmbvnr** [comment]
Let me check, but probabliy yes, the fomrs media a missing in the task form template

**Comment by kmmbvnr** [comment]
Btw, you don't need to copy who;e template content to add an additional parts. You could create a template with the same folder/name and put into extends  the same `{% viewflow/worflow/task.html %}` with only

```html
{% block extrahead %}
  {{ block.super }}
  {{ form.media }}
{% endblock %}

```

https://docs.viewflow.io/crud/templates.html

**Comment by kmmbvnr** [comment]
Fixed in 3b95c81

django-viewflow-pro==2.2.2 released
